streaming-binlog/src/main/java/org/openmrs/analytics/FhirConverter.java

```diff
@@ -127,4 +138,50 @@ GeneralConfiguration getEventsToFhirConfig(String fileName) throws IOException {
 			return generalConfiguration;
 		}
 	}
+	
+	private String getUuid(String parentTable, String parentForeignKey, String childPrimaryKeyValue, Map payload) {
+		Connection conn = null;
+		Statement stmt = null;
+		final String USER_NAME = "root";
+		final String PASS_WORD = "debezium";
+		String uuidResultFromSql = null;
+		try {
+			Class.forName("com.mysql.cj.jdbc.Driver");
+			
+			//Connecting to openmrs database
+			conn = DriverManager.getConnection(DB_URL, USER_NAME, PASS_WORD);
+			
+			//Creating statement
+			stmt = conn.createStatement();
+			
+			String sql = String.format("SELECT uuid FROM %s WHERE %s = %s", parentTable, parentForeignKey,
+			    payload.get(childPrimaryKeyValue.toString()));
+			
+			ResultSet rs = stmt.executeQuery(sql);
+			rs.next();
```

Comment:
Please check if `rs` has a next element before using it; for whatever reason the `SELECT` query may return no results.